routes/images.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const { db } = require('../conf');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  try {
+    const [images] = await db.query(`SELECT id, url, title FROM images`);
```

Comment:
There isn't a title column in db, and the route miss the description and pagesId column